### DISCUSSION TITLE
Prisma:Error Command failed with exit code 2 using query-engine-debian-openssl-1.0.x

### DISCUSSION THREAD

**Discussion by NoelHuibers** [prisma/prisma#17790] [Q&A]
I tried deploying my app on a debian server that has openssl v1.0.2g and node v18.4.0 installed. I use Next.js and starting the app (next start) does not result into any error. However when i try to call the database i get the following error:

```
prisma:error Command failed with exit code 2: /node_modules/.prisma/client/query-engine-debian-openssl-1.0.x --enable-raw-queries cli dmmf
/node_modules/.prisma/client/query-engine-debian-openssl-1.0.x: 2: /node_modules/.prisma/client/query-engine-debian-openssl-1.0.x: Syntax error: "(" unexpected
```

This is my generator schema:
```
generator client {
  provider      = "prisma-client-js"
  engineType    = "binary"
  binaryTargets = ["debian-openssl-1.0.x"]
}

```

**Comment by janpio** [comment]
1. What version of Prisma are you using?
2. I assume you having `engineType=binary` in there means you are upgrading to a newer version?
3. Why do you define the engineType manually?
4. Can you maybe share your app or a minimal reproduction of what you are seeing?

**Reply by NoelHuibers** [reply]
1. Used 4.9.0 but updated to 4.10.1 resulted in the same error.
2. No, it's because I'm running on a legacy 32-Bit System.
When trying without, I get the following message:
`prisma:error The default query engine type (Node-API, "library") is currently not supported for 32bit Node. Please set engineType = "binary" in the "generator" block of your "schema.prisma" file (or use the environment variables "PRISMA_CLIENT_ENGINE_TYPE=binary" and/or "PRISMA_CLI_QUERY_ENGINE_TYPE=binary".)`
3. I saw no possibility to compile Prisma for my binarytype on the 32-Bit server, which is why I build on a 64-Bit Docker Container of the system before deploying on the 32-Bit Linux Server.
4. Hard to do because the Error is rather Hardware specific then software specific I think.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
